### ISSUE TITLE
How to calculate the gradient of the previous layer when the gradient of the latter layer is given?

### ISSUE THREAD

**Issue by mankasto** [pytorch/pytorch#78365]
Hi, there. Can someone help me solve this problem? if the gradients of a certain layer is known, how can I use the API in torch to calculate the gradient of the previous layer?I would appreciate it if anyone could reply me in time.

**Comment by xiurobert** [comment]
You mean in a `torch.nn.Sequential`?

**Comment by dagitses** [comment]
Hi, We use github issues only for bug reports or feature requests. Please use the forum to ask questions: https://discuss.pytorch.org/
